recipes/pybammsolvers/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pybammsolvers" %}
+{% set version = "0.0.4" %}
+{% set sundials_version = "6.5.0" %}
+{% set suitesparse_version = "6.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pybammsolvers-{{ version }}.tar.gz
+    sha256: 6b76fb61e68af34b3a4803edc8020f508b10195f67857d8010e40f78cbbe2768
+  - url: https://github.com/LLNL/sundials/archive/v{{ sundials_version }}.tar.gz
+    sha256: 7557cdb8fe2a9b56fa0c259aa4e2c2dc488a1c79d482bb7943e27b7d76f6931b
+    folder: sundials
+  - url: https://github.com/DrTimothyAldenDavis/SuiteSparse/archive/refs/tags/v{{ suitesparse_version }}.tar.gz
+    sha256: 7111b505c1207f6f4bd0be9740d0b2897e1146b845d73787df07901b4f5c1fb7
+    folder: suitesparse
+
+  
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - {{ stdlib("c") }}
+    - {{ compiler('cxx') }}
+  host:
+    - python {{ python_min }}
+    - setuptools >=71
+    - casadi >=3.6.7
+    - cmake
+    - pip
+  run:
+    - python >={{ python_min }}
+    - casadi
+    - numpy <2.0
+
+test:
+  imports:
+    - pybammsolvers
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  summary: Python interface for the IDAKLU solver
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - prady0t
```

Comment:
This should be the IDAKLU maintainers group from pybamm team

Reply:
It's common for `conda-forge` feedstocks for packages to be maintained by folks other than the upstream package maintainers, so I will suggest keeping both – unless you did mean that, of course.